podman/domain/networks.py

```diff
@@ -31,7 +31,7 @@ class Network(PodmanResource):
     def id(self):  # pylint: disable=invalid-name
         """str: Returns the identifier of the network."""
         with suppress(KeyError):
-            return self.attrs["Id"]
+            return self.attrs["id"]
```

Comment:
this line is causing many troubles in the tests, I suggest reverting it

Reply:
Done.